crates/bevy_core_pipeline/src/core_3d/main_opaque_pass_3d_node.rs

```diff
@@ -16,14 +16,15 @@ use bevy_utils::tracing::info_span;
 
 use super::AlphaMask3d;
 
-/// A [`bevy_render::render_graph::Node`] that runs the [`Opaque3d`] and [`AlphaMask3d`] [`RenderPhase`].
+/// A [`bevy_render::render_graph::Node`] that runs the [`Opaque3d`]
+/// [`BinnedRenderPhase`] and [`AlphaMask3d`] [`SortedRenderPhase`]s.
 #[derive(Default)]
 pub struct MainOpaquePass3dNode;
 impl ViewNode for MainOpaquePass3dNode {
     type ViewQuery = (
         &'static ExtractedCamera,
-        &'static RenderPhase<Opaque3d>,
-        &'static RenderPhase<AlphaMask3d>,
+        &'static BinnedRenderPhase<Opaque3d>,
+        &'static SortedRenderPhase<AlphaMask3d>,
         &'static ViewTarget,
```

Comment:
AlphaMask3d should be binned too. It’s very close to opaque.

Reply:
Done.